werkzeug/debug/console.py

```diff
@@ -175,12 +175,6 @@ def runsource(self, source):
             output = ThreadedStream.fetch()
         return prompt + escape(source) + output
 
-    def runcode(self, code):
```

Comment:
I think it's important to understand what this method was overriding in Python 2. I suspect you'll now get nonsense behavior if you trigger a traceback in the Werkzeug debugger console.